app/lib/taric_synchronizer.rb

```diff
@@ -0,0 +1,137 @@
+class TaricSynchronizer
+  extend TariffSynchronizer
+
+  # 1 - does not raise an exception when record does not exist on TARIC DESTROY operation
+  #   - does not raise an exception when record does not exist on TARIC UPDATE operation
+  #   - creates new record when record does not exist on TARIC UPDATE operation
+  cattr_accessor :ignore_presence_errors
```

Comment:
I know these where inherited from the previous code but not a massive fan seeing as these are not thread safe although not sure how much an issue this considering they are more like setup code when the process is run right?

Reply:
These variables are defined under self, so I assume  they are in singleton context and would not changed unless you run the application again. So I don't think there is a thread issue here. Please make me correct if I am wrong.

Reply:
Just to weigh in here, technically its not possible for these to be read by two concurrent processes at the same time since there is effectively a redis lock/mutex in place that restricts work happening concurrently.